sormas-base/pom.xml

```diff
@@ -868,6 +868,19 @@
 				<scope>test</scope>
 			</dependency>
 
+			<dependency>
+				<groupId>de.hilling.junit.cdi</groupId>
+				<artifactId>cdi-test-core</artifactId>
+				<version>3.4.0</version>
+				<scope>test</scope>
+				<exclusions>
+					<exclusion>
```

Comment:
Don't exclude it, list the `junit-jupiter-api` also in the `dependencyManagement` with the desired junit5 version